perceptia/perceptiactl/screenshot.rs

```diff
@@ -19,26 +19,30 @@ const GLOBAL_SCREENSHOOTER: &'static str = "weston_screenshooter";
 
 // -------------------------------------------------------------------------------------------------
 
-pub fn process() {
+pub fn process(path: String) {
     println!("Taking screenshot");
-    Application::new().run(ScreenshooterConstructor::new())
+    Application::new().run(ScreenshooterConstructor::new(path))
 }
 
 // -------------------------------------------------------------------------------------------------
 
-struct ScreenshooterConstructor {}
+struct ScreenshooterConstructor {
+    path: String,
+}
 
 impl ScreenshooterConstructor {
-    fn new() -> Self {
-        ScreenshooterConstructor {}
+    fn new(path: String) -> Self {
+        ScreenshooterConstructor {
+            path,
```

Comment:
Some distributions are not up to date and still use rustc 1.16 where field shorthands give an error.